### DISCUSSION TITLE
Julia COM

### DISCUSSION THREAD

**Discussion by Ibrahim-a-Ibrahim** [dss-extensions/dss-extensions#47] [General]
This could be a general question. I am trying to connect OpenDSS with Julia via COM. However, it sounds that COM is not supported by Julia. Do you have any advise on this?

**Comment by PMeira** [comment]
If what you need is the official engine, most projects on DSS-Extensions will soon support the official OpenDSS engine using our new Oddie adaptor, which can use modern versions of EPRI's OpenDSSDirect.DLL or EPRI's new OpenDSSC library.

This link is outdated, but includes some more context: https://github.com/dss-extensions/dss_capi/tree/0.14.6a1/src/altdss_oddie

I wouldn't advise trying to use COM in Julia nowadays, especially for OpenDSS.

**Reply by Ibrahim-a-Ibrahim** [reply]
Sounds promising. I'll play around OpenDSSDirect.jl then.

**Reply by PMeira** [reply]
Trying to get a new release soon, by next week at worst. I'll update here when we have a pre-release for OpenDSSDirect.jl.

**Comment by PMeira** [comment]
Lots of issues delayed this a lot. It should finally be about in the next few days. While the documentation site is not available, if anyone wants to test this, there is a branch with the code and some notes: https://github.com/dss-extensions/OpenDSSDirect.jl/blob/dss_capi-0.15.0/docs/src/engines.md

(The upcoming release also exposes all DSS objects via the Alt API, and of course other new features from the engine.)
